src/commands/base-command.mjs

```diff
@@ -64,80 +77,141 @@ const getDuration = function (startTime) {
-    return (
-      new BaseCommand(name)
-        // If  --silent or --json flag passed disable logger
-        .addOption(new Option('--json', 'Output return values as JSON').hideHelp(true))
-        .addOption(new Option('--silent', 'Silence CLI output').hideHelp(true))
-        .addOption(new Option('--cwd <cwd>').hideHelp(true))
-        .addOption(new Option('-o, --offline').hideHelp(true))
-        .addOption(new Option('--auth <token>', 'Netlify auth token').hideHelp(true))
-        .addOption(
-          new Option(
-            '--httpProxy [address]',
-            'Old, prefer --http-proxy. Proxy server address to route requests through.',
-          )
-            .default(process.env.HTTP_PROXY || process.env.HTTPS_PROXY)
-            .hideHelp(true),
-        )
-        .addOption(
-          new Option(
-            '--httpProxyCertificateFilename [file]',
-            'Old, prefer --http-proxy-certificate-filename. Certificate file to use when connecting using a proxy server.',
-          )
-            .default(process.env.NETLIFY_PROXY_CERTIFICATE_FILENAME)
-            .hideHelp(true),
+    const base = new BaseCommand(name)
+      // If  --silent or --json flag passed disable logger
+      .addOption(new Option('--json', 'Output return values as JSON').hideHelp(true))
+      .addOption(new Option('--silent', 'Silence CLI output').hideHelp(true))
+      .addOption(new Option('--cwd <cwd>').hideHelp(true))
+      .addOption(new Option('-o, --offline').hideHelp(true))
+      .addOption(new Option('--auth <token>', 'Netlify auth token').hideHelp(true))
+      .addOption(
+        new Option('--httpProxy [address]', 'Old, prefer --http-proxy. Proxy server address to route requests through.')
+          .default(process.env.HTTP_PROXY || process.env.HTTPS_PROXY)
+          .hideHelp(true),
+      )
+      .addOption(
+        new Option(
+          '--httpProxyCertificateFilename [file]',
+          'Old, prefer --http-proxy-certificate-filename. Certificate file to use when connecting using a proxy server.',
         )
-        .option(
+          .default(process.env.NETLIFY_PROXY_CERTIFICATE_FILENAME)
+          .hideHelp(true),
+      )
+      .addOption(
+        new Option(
           '--http-proxy-certificate-filename [file]',
           'Certificate file to use when connecting using a proxy server',
-          process.env.NETLIFY_PROXY_CERTIFICATE_FILENAME,
         )
+          .default(process.env.NETLIFY_PROXY_CERTIFICATE_FILENAME)
+          .hideHelp(true),
+      )
+      .addOption(
+        new Option('--httpProxy [address]', 'Proxy server address to route requests through.')
+          .default(process.env.HTTP_PROXY || process.env.HTTPS_PROXY)
+          .hideHelp(true),
+      )
+      .option('--debug', 'Print debugging information')
+
+    // only add the `--config` or `--filter` option to commands that are workspace aware
+    if (!COMMANDS_WITHOUT_WORKSPACE_OPTIONS.has(name)) {
+      base
```

Comment:
The descriptions for these two commands should also be updated to reflect my comments in the docs (whichever version you end up using) 👍